### ISSUE TITLE
Duplicate error messages: "SRAM .bss' is not within region `ram'"

### ISSUE THREAD

**Issue by VaclavSal** [arduino/Arduino#2417]
1.5.8 on Due.

IMHO this is not a big deal, just would be nice to know "how much".
I am planning to do some simple image analysis and just checking things out for now.

It seems that the "pregnant pause" AKA initial "compiling" no verbose feedback is still there. 
Again no biggie, just an observation. 

But it seems to print duplicate messages at random when it gets printing - see attached. 

 

c:/program files/arduino_all/arduino-nightly_nov4/hardware/tools/gcc-arm-none-eabi-4.8.3-2014q1/bin/../lib/gcc/arm-none-eabi/4.8.3/../../../../arm-none-eabi/bin/ld.exe: address 0x200b8b18 of C:\DOCUME~1\Vaclav\LOCALS~1\Temp\build4303004924085407826.tmp/Audio_DUE_27.cpp.elf section `.bss' is not within region `ram'
c:/program files/arduino_all/arduino-nightly_nov4/hardware/tools/gcc-arm-none-eabi-4.8.3-2014q1/bin/../lib/gcc/arm-none-eabi/4.8.3/../../../../arm-none-eabi/bin/ld.exe: address 0x200b8b18 of C:\DOCUME~1\Vaclav\LOCALS~1\Temp\build4303004924085407826.tmp/Audio_DUE_27.cpp.elf section `.bss' is not within region `ram'
c:/program files/arduino_all/arduino-nightly_nov4/hardware/tools/gcc-arm-none-eabi-4.8.3-2014q1/bin/../lib/gcc/arm-none-eabi/4.8.3/../../../../arm-none-eabi/bin/ld.exe: address 0x200b8b18 of C:\DOCUME~1\Vaclav\LOCALS~1\Temp\build4303004924085407826.tmp/Audio_DUE_27.cpp.elf section `.bss' is not within region `ram'
c:/program files/arduino_all/arduino-nightly_nov4/hardware/tools/gcc-arm-none-eabi-4.8.3-2014q1/bin/../lib/gcc/arm-none-eabi/4.8.3/../../../../arm-none-eabi/bin/ld.exe: address 0x200b8b18 of C:\DOCUME~1\Vaclav\LOCALS~1\Temp\build4303004924085407826.tmp/Audio_DUE_27.cpp.elf section `.bss' is not within region `ram'
c:/program files/arduino_all/arduino-nightly_nov4/hardware/tools/gcc-arm-none-eabi-4.8.3-2014q1/bin/../lib/gcc/arm-none-eabi/4.8.3/../../../../arm-none-eabi/bin/ld.exe: address 0x200b8b18 of C:\DOCUME~1\Vaclav\LOCALS~1\Temp\build4303004924085407826.tmp/Audio_DUE_27.cpp.elf section `.bss' is not within region `ram'
collect2.exe: error: ld returned 1 exit status
c:/program files/arduino_all/arduino-nightly_nov4/hardware/tools/gcc-arm-none-eabi-4.8.3-2014q1/bin/../lib/gcc/arm-none-eabi/4.8.3/../../../../arm-none-eabi/bin/ld.exe: address 0x200b8b18 of C:\DOCUME~1\Vaclav\LOCALS~1\Temp\build4303004924085407826.tmp/Audio_DUE_27.cpp.elf section `.bss' is not within region `ram'
collect2.exe: error: ld returned 1 exit status
Error compiling.
Error compiling.

**Comment by facchinm** [comment]
Still happens on 1.6.6
Could be solved together with #766 and #3568 by making both "verify" and "upload" buttons stateful; the alternate state would ideally stop (kill) the spawned process (compiler or programmer)

**Comment by ffissore** [comment]
Merged into #3771

**Comment by VaclavSal** [comment]
I would like to add a similar problem, but not sure if I can really describe it. 
I  suspect it has something to do with my OS "double clicking" ( wireless mouse) also.
I am getting status message  "Uploading" but I can see no "...erase.. reset cpu " messages and no RX/TX  COM LED flashing either. ( Using Due) 
Sometime I get something about "unable to access COMx " in red as status or "unable to access such and such file " , also in read , but no Upload. 
The worst part , I just discovered recently - I can change stuff in editor, click "Upload " , it executes upload but the edited file does not get updated! ( I have not tried to analyze the compiler output to see if the file I was editing is in fact recompiled) .
This got so bad I had to reload another version of 1.6.6 ( nightly) to fix it, but it is still happening occasionally. 
The weird part is - sometime the editor jumps from current file / tab to FIRST tab and highlights ( in pink) the first line in it as an error(?) I think that also indicates that multiple mouse clicks are detected by IDE. 
I be happy to provide details if necessary or pay attention to some specifics, but so far I am just an user, not an IDE developer.
